pkg/report/linux.go

```diff
@@ -575,7 +575,7 @@ var linuxOopses = []*oops{
 				fmt:   "BUG: sleeping function called from invalid context %[1]v",
 			},
 			{
-				title: compile("BUG: using __this_cpu_([a-z_]+)\\(\\) in preemptible .*(?:.*\\n){0,30}Call Trace:\\n" + stacktraceRe("dump_stack", "preemption", "preempt")),
+				title: compile("BUG: using __this_cpu_([a-z_]+)\\(\\) in preemptible .*(?:.*\\n){0,30}?Call Trace:\\n" + stacktraceRe("dump_stack", "preemption", "preempt")),
```

Comment:
There seems to be a similar case in "trying to register non-static key" and maybe more.

Reply:
Fixed them as well.